projects/dynamicforms/src/components/currency/currency.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Injector } from '@angular/core';
+import { Observable } from 'rxjs';
+import { AbstractFormComponent } from '../abstract-form/abstract-form.component';
+import { DynamicFormsConfig } from '../../core/config/form-config';
+import { CurrencyService, LanguageService } from '@spartacus/core';
+import { FormService } from '../../core/services/form/form.service';
+
+@Component({
+  selector: 'cx-currency',
+  templateUrl: './currency.component.html',
+})
+export class CurrencyComponent extends AbstractFormComponent {
```

Comment:
Can you please give JSON representation for currency component?

Reply:
Currently, I changed only `fieldType` from `input` to `currency`. Who is working on the back should add format regex attribute in JSON.